cinema/serializers.py

```diff
@@ -1 +1,82 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
```

Comment:
Consider adding type annotations to the serializer classes to improve code readability and maintainability. For example, you can specify the type of the Meta class as `class Meta: ...`.